ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -657,3 +657,37 @@ def test_paddle_cond(*, dtype_and_x, p, on_device, fn_tree, frontend, test_flags
         atol=1e-5,
         p=p,
     )
+
+# dot
+@handle_frontend_test(
+    fn_tree="paddle.tensor.linalg.dot", 
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=(1,),
```

Comment:
Is that necessary?

Reply:
I think having it 1 dimensional is necessary for the a vector dot product.

Reply:
I think this is a scaler shape not a 1-dimentsional one, also paddle support 2D shapes as inputs apparently according to their [docs](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/dot_en.html).